.github/workflows/ci.yml

```diff
@@ -1062,8 +1062,6 @@ jobs:
   # Jobs that cross-compile IREE for other platforms
   ##############################################################################
 
-  # emscripten is not in the test matrix because it is set as
-  # postsubmission-only.
   cross_compile_and_test:
```

Comment:
This name is pretty long. How do we feel about shortening it to `cross_compile` or something like `platform_matrix`? @pzread / @hcindyl 

(context: https://github.com/iree-org/iree/pull/10746)

Reply:
@pzread commented on https://github.com/iree-org/iree/pull/10746#discussion_r997682592 to put the suffix `_test` in the name. I am okay either way. Will defer to you can Jerry to decide the name.

Reply:
The names end up pretty long in these views, so it's hard to which platform in the matrix is which:

PR checks:
![image](https://user-images.githubusercontent.com/4010439/217088307-79f9bd00-fe62-453b-ae0a-758870a8d1d3.png)

runs/id/
![image](https://user-images.githubusercontent.com/4010439/217088451-a695f740-4c89-4923-ab9f-8c1c0a96db9d.png)

Reply:
I have a low-priority item to rework the naming of everything. We had a Discord discussion starting here: https://discord.com/channels/689900678990135345/689957613152239638/1054588881045037056. I agree that the length is a problem here. That view being readable is pretty important :-) Maybe we can override `name` using the matrix? I'm not sure if that's possible. So far we've just used the job id as the name (I think it's simpler that way), but we do have the ability to configure it. Then it could be `android_armv8.2-a_arm64-v8a` and such (no docker container name either)

Reply:
That will be nice. We can also steer towards using job matrices for more configurations (that will improve grouping in the UI and reduce duplication in the .yml).

Reply:
This shouldn't block turning on emscripten though. Let's address that later